src/components/internal-header/_internal-header.scss

```diff
@@ -12,6 +12,13 @@
   }
 }
 
+@media (forced-colors: active) {
+  svg {
+    forced-color-adjust: none;
+    color: canvastext;
```

Comment:
Had a look at this again this morning and because the internal banner icon is a child of a link this should be `linktext`